### DISCUSSION TITLE
Missing configuration parameters after the last update

### DISCUSSION THREAD

**Discussion by ldiegos** [txubelaxu/esphome-jk-bms#21] [General]
Hi txubelaxu,

Sorry, but after your last updates/improvements, there are missing some of the parameters, also, the configuration and the dashboard breaks...

You made a lot of changes in the config parameters and was a little crazy for me 👍 

Could be feasible for you, to start managing releases? 
So anyone, like me, that need to recompile again your fantastic module for homeassistant can update everything with the same release, despite being old, but working for any of us.  And leaving up to us, to migrate to newer versions.

So in the configuration file for the ESP32 we can attach the release, instead of the main and while the release is active, we could use it:
From this: **external_components_source: github://txubelaxu/esphome-jk-bms@main**
to this: **external_components_source: github://txubelaxu/esphome-jk-bms_@relase2.1.4.78_**

Thank you so much for your time and wonderfull RS485 module

![image](https://github.com/user-attachments/assets/57153ce0-4fbd-4191-a341-d0fad9839746)

![image](https://github.com/user-attachments/assets/a015a2c4-72b4-4859-99fb-0973a720b648)

**Comment by txubelaxu** [comment]
It is because I have moved from "sensor" to "number". This way the values is WRITABLE now. Not from that panel because is not clickable. But the parameters are present as numbers in the HA device.

You have here the dashboard: https://github.com/txubelaxu/esphome-jk-bms/blob/main/components/jk_rs485_bms/home_assistant_dashboards/0bms_master_and_2bms_slaves.yaml

**Comment by ldiegos** [comment]
i know i know, i found out yesterday late night... :)
